web/modules/custom/rdf_entity/rdf_draft/rdf_draft.module

```diff
@@ -60,7 +71,7 @@ function rdf_draft_entity_operation(EntityInterface $entity) {
   if ($storage  instanceof RdfEntitySparqlStorage) {
     $definitions = $storage->getGraphDefinitions();
     foreach ($definitions as $name => $definition) {
-      if ($entity->hasLinkTemplate('rdf-draft-' . $name)) {
+      if ($entity->hasLinkTemplate('rdf-draft-' . $name) && $entity->hasGraph($name)) {
```

Comment:
The templated is not set for an entity but for an entity type. Example: `rdf_entity` entity type has the template but specific entity, say a solution, of type `rdf_entity` might not have a `draft` version/graph. In that case we don't want to add the operation/link "View draft". There's no draft.